apps/epic-react/src/templates/lesson-template.tsx

```diff
@@ -221,8 +221,10 @@ const ExerciseTemplate: React.FC<{
               >
                 <div className={isTheaterMode ? 'col-span-4' : ''}>
                   <div className="flex w-full items-center justify-between py-5 sm:py-6">
-                    {epicReactModule.github?.repo && (
+                    {epicReactModule.github?.repo ? (
                       <RepositoryLink codeUrl={epicReactModule.github.repo} />
+                    ) : (
+                      <div />
```

Comment:
I haven't seen an empty div be used like this in a ternery, usually `null`. Is there a reason you aren't using `null`?